score-data-transfer/tests/checker.nf

```diff
@@ -29,7 +29,7 @@
 /* this block is auto-generated based on info from pkg.json where   */
 /* changes can be made if needed, do NOT modify this block manually */
 nextflow.enable.dsl = 2
-version = '0.4.0'
+version = '0.5.0'
```

Comment:
if you update wfpm cli to the latest version `0.7.8`, the version string in checker will be updated when you run `wfpm nextver` command, so you won't need to do this manually